From 7f1c9d7608d3df90fc7433fb5b829c2b8d6f22b1 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sun, 19 Aug 2018 08:53:49 -0600
Subject: [PATCH 3/6] DO NOT MERGE: klte-common: Requisite bring-up BS change

Currently:
* Permissive
  Note: Other kernel commandline updates are necessary to
  temporarily free up line space
* Kill s3fwrn5 NFC

Change-Id: Iec4e6927e019c9ec91cc16d3eeacb0990ffff6f5
---
 BoardConfigCommon.mk   | 3 ++-
 nfc/s3fwrn5/board.mk   | 2 +-
 nfc/s3fwrn5/product.mk | 6 +++---
 3 files changed, 6 insertions(+), 5 deletions(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 2f35c1a..7faa9ac 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -40,7 +40,8 @@ DEVICE_MANIFEST_FILE += $(COMMON_PATH)/manifest.xml
 
 # Kernel
 BOARD_KERNEL_BASE := 0x00000000
-BOARD_KERNEL_CMDLINE := console=null androidboot.hardware=qcom user_debug=31 msm_rtb.filter=0x37 ehci-hcd.park=3 zcache.enabled=1 zcache.compressor=lz4 maxcpus=1
+BOARD_KERNEL_CMDLINE := androidboot.hardware=qcom user_debug=31 msm_rtb.filter=0x37 ehci-hcd.park=3 zcache.enabled=1 zcache.compressor=lz4 maxcpus=1
+BOARD_KERNEL_CMDLINE += androidboot.selinux=permissive
 BOARD_KERNEL_IMAGE_NAME := zImage
 BOARD_KERNEL_PAGESIZE := 2048
 BOARD_KERNEL_SEPARATED_DT := true
diff --git a/nfc/s3fwrn5/board.mk b/nfc/s3fwrn5/board.mk
index 4c8a742..e88095e 100644
--- a/nfc/s3fwrn5/board.mk
+++ b/nfc/s3fwrn5/board.mk
@@ -15,4 +15,4 @@
 #
 
 # NFC
-include $(COMMON_PATH)/nfc/board.mk
+#include $(COMMON_PATH)/nfc/board.mk
diff --git a/nfc/s3fwrn5/product.mk b/nfc/s3fwrn5/product.mk
index d4f9497..7bbf86b 100644
--- a/nfc/s3fwrn5/product.mk
+++ b/nfc/s3fwrn5/product.mk
@@ -15,11 +15,11 @@
 #
 
 # NFC
-$(call inherit-product, device/samsung/klte-common/nfc/product.mk)
+#$(call inherit-product, device/samsung/klte-common/nfc/product.mk)
 
-PRODUCT_PACKAGES += \
+#PRODUCT_PACKAGES += \
     android.hardware.nfc@1.0-impl
 
-PRODUCT_COPY_FILES += \
+#PRODUCT_COPY_FILES += \
     device/samsung/klte-common/nfc/s3fwrn5/libnfc-sec.conf:$(TARGET_COPY_OUT_VENDOR)/etc/libnfc-brcm.conf \
     device/samsung/klte-common/nfc/s3fwrn5/libnfc-sec-hal.conf:$(TARGET_COPY_OUT_VENDOR)/etc/sec-nfc.conf
-- 
2.17.1

